Sources/BraintreeCore/Analytics/BTAnalyticsMetadata.swift

```diff
@@ -1,89 +0,0 @@
-// JIRA - DTBTSDK-2683
-struct BTAnalyticsMetadata {
-
-    // MARK: Metadata Properties
-
-    static let platform: String = "iOS"
-    static let platformVersion: String = UIDevice.current.systemVersion
-    static let sdkVersion: String = BTCoreConstants.braintreeSDKVersion
-    static let merchantAppID: String = Bundle.main.infoDictionary?[kCFBundleIdentifierKey as String] as? String ?? ""
-    static let merchantAppVersion: String = Bundle.main.infoDictionary?[kCFBundleVersionKey as String] as? String ?? ""
-    static let merchantAppName: String = Bundle.main.infoDictionary?[kCFBundleNameKey as String] as? String ?? ""
-    static let deviceManufacturer: String = "Apple"
-    static let iOSDeviceName: String = UIDevice.current.name
-    static let iOSSystemName: String = UIDevice.current.systemName
-
-    // MARK: Metadata Computed Properties
-
-    static var deviceModel: String {
-        var systemInfo = utsname()
-        uname(&systemInfo)
-        let machineMirror = Mirror(reflecting: systemInfo.machine)
-        let identifier = machineMirror.children.reduce("") { identifier, element in
-            guard let value = element.value as? Int8, value != 0 else { return identifier }
-            return identifier + String(UnicodeScalar(UInt8(value)))
-        }
-        return identifier
-    }
-
-    static var iOSPackageManager: String {
-        #if COCOAPODS
-            return "CocoaPods"
-        #elseif SWIFT_PACKAGE
-            return "Swift Package Manager"
-        #else
-            return "Carthage or Other"
-        #endif
-    }
-
-    static var isSimulator: Bool {
-        #if targetEnvironment(simulator)
-            return true
-        #else
-            return false
-        #endif
-    }
-
-    static var isVenmoInstalled: Bool {
-        guard let venmoURL = URL(string: "com.venmo.touch.v2://x-callback-url/vzero/auth") else { return false }
-        return UIApplication.shared.canOpenURL(venmoURL)
-    }
-
-    static var dropInVersion: String {
-        var dropInVersion: String = ""
-        let localizationBundlePath = Bundle.main.path(forResource: "Braintree-UIKit-Localization", ofType: "bundle")
-        if localizationBundlePath != nil, let localizationBundlePath {
-            let localizationBundle = Bundle(path: localizationBundlePath)
-            // 99.99.99 is the version specified when running the Demo app for this project.
-            // We want to ignore it in this case and not return a version.
-            if localizationBundle != nil && localizationBundle?.infoDictionary?["CFBundleShortVersionString"] as? String != "99.99.99" {
-                dropInVersion = localizationBundle?.infoDictionary?["CFBundleShortVersionString"] as? String ?? ""
-            }
-        }
-        return dropInVersion
-    }
-
-    // MARK: - Construct Metadata
-
-    static var metadata: [String: Any] {
-        [
-            "platform": platform,
-            "platformVersion": platformVersion,
```

Comment:
q: does this remove OS version tracking?

Reply:
Client OS is still sent [here](https://github.com/braintree/braintree_ios/blob/67cb8db8fd26f0fc294c5e0c41ba0326166c8c79/Sources/BraintreeCore/Analytics/FPTIBatchData.swift#L107), in the analytics metadata.

We were sending our "AnalyticsMetadata" in every single API request to the gateway, for example tokenizing a card. We were sending things like, `isSimulator`, `iosPackageManager`, etc - things not relevant to tokenization. I am gonna guess that it started off that way since a few things did overlap with analytics metadata values, like platform and SDK version.

Let me know if that makes sense.

Reply:
Got it yeah I was confused because it looks like a pure deletion in the diff. I wasn't aware it was duplication.